learn/advanced/asynchronous_operations.md

```diff
@@ -252,70 +197,19 @@ This command returns tasks two at a time, starting from task `uid` `10`.
 {
   "results": [
-    {
-      "uid": 10,
-      "indexUid": "elements",
-      "status": "succeeded",
-      "type": "indexCreation",
-      "canceledBy": null,
-      "details": {
-        "primaryKey": null
-      },
-      "error": null,
-      "duration": "PT0.006034S",
-      "enqueuedAt": "2022-06-20T13:41:42.446908Z",
-      "startedAt": "2022-06-20T13:41:42.447477Z",
-      "finishedAt": "2022-06-20T13:41:42.453511Z"
-    },
-    {
-      "uid": 9,
-      "indexUid": "particles",
-      "status": "succeeded",
-      "type": "indexCreation",
-      "canceledBy": null,
-      "details": {
-        "primaryKey": null
-      },
-      "error": null,
-      "duration": "PT0.007317S",
-      "enqueuedAt": "2022-06-20T13:41:31.841575Z",
-      "startedAt": "2022-06-20T13:41:31.842116Z",
-      "finishedAt": "2022-06-20T13:41:31.849433Z"
-    }
+    …
   ],
   "limit": 2,
   "from": 10,
   "next": 8
 }
 ```
 
-To view the next page of results, you would repeat the same query, replacing the value of `from` with the value of `next`:
+To view the next set of results, you would repeat the same query, replacing the value of `from` with the value of `next`:
 
 <CodeSamples id="get_all_tasks_paginating_2" />
 
-When the returned value of `next` is `null`, you have reached the final page of results.
-
-## Task workflow
-
-1. When you make an [asynchronous request](#which-operations-are-async), Meilisearch puts it in the task queue, sets the task's `status` to `enqueued` and returns a [summarized `task` object](/learn/advanced/asynchronous_operations.md#summarized-task-objects)
-2. When your task reaches the front of the queue, Meilisearch begins working on it and changes the request `status` to `processing`
-3. You can cancel a task while it is in the `enqueued` or `processing` states. If canceled, it will have the `canceled` status  
-4. Once the task has completed processing, Meilisearch marks it as `succeeded`, if it was successful, or `failed`, if there was an error
-5. Tasks marked as `succeeded`, `failed`, or `canceled` are not automatically deleted and will remain visible in [the task list](/reference/api/tasks.md#get-tasks). They can be deleted manually using the [delete tasks route](/reference/api/tasks.md#delete-tasks)
-
-### Task priority
-
-Tasks are processed in the order they were enqueued, with the exception of `taskCancelation` which is processed in reverse order.
-
-`taskCancelation`, `taskDeletion`, `snapshotCreation`, and `dumpCreation` are prioritized over all other tasks in the queue. Their task `uid`s still reflect when they were enqueued relative to other tasks.
-
-The following list shows different task types in decreasing order of priority:
-
-1. `taskCancelation`
-2. `taskDeletion`
-3. `snapshotCreation`
-4. `dumpCreation`
-5. All other task types in the order they were enqueued
+When the returned value of `next` is `null`, you have reached the final set of results.
 
 ## Terminate Meilisearch while a task is being processed
```

Comment:
Do you feel that this section is necessary and helpful? Does it make sense to exist in this guide, versus in the FAQ?